llvm/tools/llvm-readtapi/llvm-readtapi.cpp

```diff
@@ -255,16 +255,21 @@ static void stubifyDirectory(const StringRef InputPath, Context &Ctx) {
     if (EC)
       reportError(IT->path() + ": " + EC.message());
 
-    // Skip header directories (include/Headers/PrivateHeaders) and module
```

Comment:
How is this changing readtapi's behavior?

Reply:
My suspicion of why the windows bot failed on the `stubify-delete` test is because the conditional includes the path style, which likely differs for a non-unix system. I changed it to omit "/"